client/src/view/components/Header/Header.test.js

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from './Header';
+import {Provider} from "react-redux";
+import store from "../../../redux/store/store";
+import React from "react";
+import Nav from "./Nav/Nav";
+import CatalogNav from "./CatalogNav/CatalogNav";
+
+const mobileHeader =
+    <>
+        <Nav/>
+        <CatalogNav/>
+    </>
+
+const MockHeader = () => {
+  return (
+      <BrowserRouter>
+        <Provider store={store}>
+          <Header/>
+        </Provider>
+      </BrowserRouter>
+  )
+}
+
+const MobileHeader = () => {
+    return (
+        <BrowserRouter>
+            <Provider store={store}>
+                {mobileHeader}
+            </Provider>
+        </BrowserRouter>
+    )
+}
+
+describe("Header component", () => {
+    it('renders Header without crashing', () => {
```

Comment:
Please use "test" keyword instead of "it"

Reply:
+